org.eclipse.xtext.tests/src/org/eclipse/xtext/generator/trace/node/GeneratorNodePerformanceTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2018 TypeFox (https://typefox.io) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.generator.trace.node;
+
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.xtext.generator.trace.LocationData;
+import org.eclipse.xtext.generator.trace.SourceRelativeURI;
+import org.eclipse.xtext.xbase.lib.StringExtensions;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+
+/**
+ * @author Dennis Huebner - Initial contribution and API
+ */
+public class GeneratorNodePerformanceTest {
+	private GeneratorNodeExtensions exts = new GeneratorNodeExtensions();
+
+//	private static final int numberOfLines = 800_000 // Before x sec - after 4.6 sec
+//	private static final int numberOfLines = 80_000 // Before 89 sec - after 0.43 sec
+	private static final int numberOfLines = 40_000; // Before 20 sec - after
+														// 0.32 sec
```

Comment:
Merge with comment above, it looks very weird floating by itself here

Reply:
autoformatter is arggg

Reply:
Yeah, for some reason the default formatting length for comments is 80 chars. We've changed it to 120 in our codebase, same as for java lines.
Not sure why you'd want a different allowed length for your comments than for your code